### PR TITLE
Add comment category and body into stream

### PR DESCRIPTION
**Pull request by marten**

### CODE REVIEW

File: app/models/concerns/comment/publishing.rb [hunk 1]
```diff
@@ -24,7 +24,9 @@ def to_event_stream
         focus_id: focus_id,
         focus_type: focus_type,
         project_id: project_id,
+        category: category,
```
Comment (parrish) [inline app/models/concerns/comment/publishing.rb]: FWIW, this is currently an unused field

File: app/models/concerns/comment/publishing.rb [hunk 2]
```diff
@@ -24,7 +24,9 @@ def to_event_stream
...
         section: section,
+        body: body,
```
Comment (parrish) [inline app/models/concerns/comment/publishing.rb]: Do we not have to be concerned about the data size here?  Some comments get really large

Comment (marten) [inline app/models/concerns/comment/publishing.rb]: We're limited to 1MB per stream message, but so far our biggest comment is less than 10kb.

File: app/models/concerns/comment/publishing.rb (hunk 1 above)
Comment (marten) [inline app/models/concerns/comment/publishing.rb]: Ah, okay, I'll get rid of this one again then.

**Pull request merged**